editor/index.js

```diff
@@ -107,7 +108,9 @@ export function createEditorInstance( id, post, editorSettings = DEFAULT_SETTING
 						onUndo: undo,
 					}, store.dispatch ) }
 				>
-					<Layout settings={ editorSettings } />
+					<EditorSettingsProvider settings={ editorSettings }>
```

Comment:
I wonder if we can start thinking of a nicer way to compose all these providers...